app/Http/Middleware/ScopeBouncer.php

```diff
@@ -37,7 +37,7 @@ public function handle($request, Closure $next)
         $tenantId = 0;
         $authUser = $request->user();
 
-        if ($authUser && $authUser->publicAdministrations->isNotEmpty()) {
+        if ($authUser && session()->has('tenant_id')) {
```

Comment:
Io qua pensavo di gestire il remoto caso di un utente che viene cancellato da una pa mentre è autenticato. Ha il tenant id in sessione ma in realtà non fa più parte della PA.

Reply:
Si tratta di una semplificazione di quanto avevo introdotto in f325dfe96eb73bca1f6b9ebdeab76560de300629.
Dato che dopo questo middleware viene applicato `SelectTenant`, il caso d'uso dell'utente cancellato è coperto dal codice: https://github.com/AgID/wai-portal/blob/0af0dd4383a587c78c7f543db8fdbaa433c11570/app/Http/Middleware/SelectTenant.php#L35-L39
Questo metodo è anche più completo perché con `$authUser->publicAdministrations->isNotEmpty()` veniva coperto solo il caso "utente cancellato da tutte le PA".